### ISSUE TITLE
Error in tests ? - should pluck template literals from gql by default even if not imported from graphql-tag

### ISSUE THREAD

**Issue by jide** [ardatan/graphql-toolkit#351]
I was scanning tests and saw this : https://github.com/ardatan/graphql-toolkit/blob/master/packages/graphql-tag-pluck/tests/graphql-tag-pluck.test.ts#L625

```js
  it ('should pluck template literals from gql by default even if not imported from graphql-tag', async () => {
    const file = await createTmpFile({
      unsafeCleanup: true,
      template: '/tmp/tmp-XXXXXX.js',
    })

    writeFileSync(file.name, freeText(`
      import gql from 'graphql-tag'
```

Shouldn't the test use an import different from `graphql-tag` ?

**Comment by ardatan** [comment]
We already have tests for other modules.
See;
https://github.com/ardatan/graphql-toolkit/blob/6a1ef103c7e40b65c473e5085adcbee61dc46eb2/packages/graphql-tag-pluck/tests/graphql-tag-pluck.test.ts#L929
Is that what you asked? Or something else because the title has error?

**Comment by jide** [comment]
The test description mentions testing "even if not imported from graphql-tag" although it does not seem to test that. No big deal anyway :)

**Comment by ardatan** [comment]
GraphQL Tag Pluck exports type definitions not only by looking at `gql` tags. It looks for `gql` tags imported from specific modules. 
https://github.com/ardatan/graphql-toolkit/blob/6d1b3021d348144e8f447bdc0eceb0d6ef21b16c/packages/graphql-tag-pluck/src/visitor.ts#L5

**Comment by jide** [comment]
Yes I understood that, it's just that the test only tests for `import gql from 'graphql-tag'`. But maybe it's on purpose.

**Comment by ardatan** [comment]
@jide Oh you're right. Fixed in #390 :) Thank you!

**Comment by ardatan** [comment]
Available in v0.8.1! :)

**Comment by jide** [comment]
Hehe ;)
